docs/content/configuration/coordinator.md

```diff
@@ -22,6 +22,7 @@ The coordinator node uses several of the global configs in [Configuration](../co
 
 |Property|Description|Default|
 |--------|-----------|-------|
+|`druid.coordinator.beOverlord`|Boolean value for whether this coordinator node should act like an overlord as well. This configuration allows users to simplify a druid cluster by not having to deploy any standalone overlord nodes. If set to true, then make sure `druid.selectors.indexing.serviceName` is correctly set to coordinator service name everywhere and all the overlord configuration must be added to the coordinator node configuration.|true|
```

Comment:
that sounds good to enable rolling deploy, will explore how to enable that.
we probably will have to create another ServiceAnnouncer at coordinator using druid.selectors.indexing.serviceName property . however, that would mean users would need to correctly specify `druid.selectors.indexing.serviceName` at coordinators also which wasn't necessary till now.